From 7eaf1be922d9f30e1905e38da5e71eb02b054be9 Mon Sep 17 00:00:00 2001
From: yxj <yxj@rock-chips.com>
Date: Thu, 29 Mar 2012 19:11:32 +0800
Subject: [PATCH] rk30 fb: fix a bug in  init_lcdc_device_driver

	un init load_screen for dev_drv->load_scren before
---
 drivers/video/rockchip/rk_fb.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/video/rockchip/rk_fb.c b/drivers/video/rockchip/rk_fb.c
index 78ba1805afec..0e2aa8e23a9a 100644
--- a/drivers/video/rockchip/rk_fb.c
+++ b/drivers/video/rockchip/rk_fb.c
@@ -733,6 +733,7 @@ int init_lcdc_device_driver(struct rk_lcdc_device_driver *def_drv,
 	dev_drv->pan_display = def_drv->pan_display;
 	dev_drv->suspend = def_drv->suspend;
 	dev_drv->resume = def_drv->resume;
+	dev_drv->load_screen = dev_drv->load_screen;
 	
 	return 0;
 }
-- 
2.35.3

